telescope/query.py

```diff
@@ -169,15 +176,23 @@ def _create_query_string(self, mlab_project='ndt'):
     log_times_joined = ' OR\n\t'.join(self._conditional_dict['log_time'])
     conditional_list_string += '\n\tAND ({log_times})'.format(
         log_times=log_times_joined)
-
-    server_ips_joined = ' OR\n\t\t'.join(self._conditional_dict['server_ip'])
-    conditional_list_string += '\n\tAND ({server_ips})'.format(
-        server_ips=server_ips_joined)
-
-    client_ips_joined = ' OR\n\t\t'.join(
-        self._conditional_dict['client_network_block'])
-    conditional_list_string += '\n\tAND ({client_ips})'.format(
-        client_ips=client_ips_joined)
+    
+    if self._conditional_dict.has_key('server_ips'):
+        server_ips_join = " OR\n\t\t".join(self._conditional_dict['server_ips'])
+        conditional_list_string += "\n\tAND ({server_ips})".format(
+            server_ips = server_ips_join)
+    
+    if self._conditional_dict.has_key('client_ips'):
+        client_ips_join = " OR\n\t\t".join(self._conditional_dict['client_ips'])
+        conditional_list_string += "\n\tAND ({client_ips})".format(
+            client_ips = client_ips_join)
+    
+    if self._conditional_dict.has_key('client_country'):
+        conditional_list_string += "\n\tAND ({conditional})".format(
+            conditional = self._conditional_dict['client_country'])
+    if self._conditional_dict.has_key('server_country'):
```

Comment:
Yes, it was in a previous version but I didn't fully implement it here because I am not sure its necessary at this time and don't want to overexpand Telescope yet. I'll remove this and if it is of value, we could revisit the issue later.